client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -48,13 +49,14 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 		};
 	}
 
-	handleAddBusinessAddress = () => {
+	handleSubmit = () => {
```

Comment:
I think you this one needs to be added as a `onSubmit` to the form.

Reply:
ofc, thanks.